examples/plugins/kubectl/plugin.yaml

```diff
@@ -0,0 +1,7 @@
+name: "kubectl"
+version: "0.1.0"
+usage: conftest kubectl (TYPE[.VERSION][.GROUP] [NAME] | TYPE[.VERSION][.GROUP]/NAME).
+description: |-
+  A Kubectl plugin for using Conftest to test objects in Kubernetes using Open Policy Agent.
```

Comment:
This is actually now a conftest plugin for using kubectl :)

Reply:
Haha good catch!